tests/layer_tests/tensorflow_tests/test_tf_Div.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+import tensorflow as tf
+from common.tf_layer_test_class import CommonTFLayerTest
+
+
+class TestDiv(CommonTFLayerTest):
+    def _prepare_input(self, inputs_info):
+        assert 'x' in inputs_info
+        assert 'y' in inputs_info
+        x_shape = inputs_info['x']
+        y_shape = inputs_info['y']
+        inputs_data = {}
+        # generate x and y
+        inputs_data['x'] = np.random.randint(-10, 10, x_shape).astype(self.input_type)
+        inputs_data['y'] = np.random.randint(1, 10, y_shape).astype(self.input_type)
```

Comment:
Are there any limitation that "y" should be positive? Can we change to `np.random.randint(-10, 10, y_shape)`

Reply:
When the denominator is zero, the output will be inf. Both functions correctly return inf. However, the function that compares them returns false, which causes the test to fail. The `allclose` function in the `common_utils.py` file returns false for two equal values that include inf. Therefore, I excluded zero from the denominator values for the test. Of course, I will include negative values as well.

Reply:
Okay, then let's create positive array and multiply on random sign to have both positive and negative non-zero values.
`np.random.randint(1, 10, y_shape)*np.random.choice([-1,1], y_shape)`